webui/src/styles/checklist.scss

```diff
@@ -0,0 +1,185 @@
+body {
+  overflow-x: hidden;
+}
+
+.checklist {
+  color: #343a40;
+
+  & .checklist-title {
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  & .checklist-description {
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  & .checklist-item-header {
+    height: 96px;
+    padding: 32px 0;
+    box-sizing: border-box;
+    border: none;
+
+    & > button {
+      padding: 0;
+      color: #343a40;
+      background-color: #fff;
+      border: none;
+
+      &:focus {
+        outline: 0;
+        box-shadow: none;
+      }
+      &:not(.collapsed) {
+        color: #343a40;
+      }
+    }
+  }
+
+  & .checklist-item-header-text {
+    font-size: 14px;
+    font-weight: 400;
+    margin-left: 16px;
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  & .accordion-body {
+    padding-top: 0;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  & .checklist-item-text {
+    margin-bottom: 8px;
+  }
+
+  & .cta-button {
+    display: block;
+    font-size: 14px;
+  }
+}
+
+.checklist-slider {
+  bottom: initial;
+  box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.25);
+  border-radius: 8px 0 0 8px;
+  border: none;
+
+  & .checklist-slider-body {
+    flex-grow: 0;
+    padding: 32px 32px 0 32px;
+  }
+}
+
+.checklist-slider-button {
+  padding: 16px 24px;
+  width: 192px;
+  height: 56px;
+
+  &:active {
+    background-color: #343a40 !important;
+    color: #000 !important;
+  }
+
+  &.checklist-button-external {
+    transform: translateY(-50%) rotate(-90deg);
+    position: absolute;
+    right: -68px;
+    top: 50%;
+    background-color: #343a40;
+    color: #fff;
+    border-radius: 8px 8px 0 0;
+    border: none;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+
+    &:hover {
+      animation-name: pullOut;
+      animation-duration: 1.4s;
+      animation-fill-mode: both;
+    }
+  }
+
+  &.checklist-button-internal {
+    border-radius: 8px 8px 0 0;
+    background-color: #343a40;
+    color: #fff;
+    position: absolute;
+    left: -125px;
+    top: 50%;
+    transform: translateY(-50%) rotate(-90deg);
+    z-index: -1;
+    border: none;
+    font-size: 16px;
+    font-weight: 500;
+  }
+}
+
+.accordion-button:not(.collapsed)::after {
+  background-image: var(--bs-accordion-btn-icon);
+}
+
+@keyframes pullOut {
```

Comment:
Not a fan of animations in working GUIs: they are often poor a11y.  Where does it appear?  Because spinouts can distract people with poor vision or various other conditions.

Here in particular it seems like "flashing" according to [understanding pause-stop-hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html).  **I am not an expert**, but this seems rather unnecessary behaviour.

Reply:
No animation is "necessary"; I think we agree on that. Since this is CSS animation, it can be disabled by browser preference, and many assistive technologies indeed do so.  

In this specific case, it's a subtle animation only triggered when hovering over the checklist tab. This means it's not automatic and doesn't last more than 5 seconds. Moreover, it isn't used to convey any unique information which isn't available to users of assistive technologies.
It was intentionally kept super subtle and the least distracting it could be. You can see how it looks below.

https://github.com/treeverse/lakeFS/assets/110764839/aa8e607b-e9e8-4995-b5dd-49c50bd02e3d

Reply:
Well if it's standard we can turn it off.  Thanks! :-)

Reply:
> Well if it's standard we can turn it off. Thanks! :-)

I'd like to rephrase!

I'm glad that it's standard.  That means users can turn it off, and order is restored to the world.  Thanks!